client/src/AppBundle/Service/DocumentSyncService.php

```diff
@@ -204,25 +216,25 @@ private function handleDocumentStatusUpdate(Document $document, string $status,
 
     private function handleReportSubmissionUpdate(int $reportSubmissionId, string $uuid)
     {
-        try {
-            return $this->restClient->put(
-                sprintf('report-submission/%s', $reportSubmissionId),
-                json_encode(['data' => ['uuid' => $uuid]])
-            );
-        } catch (Throwable $exception) {
-            return $exception;
-        }
+        return $this->restClient->apiCall(
+            'put',
+            sprintf('report-submission/%s/update-uuid', $reportSubmissionId),
+            json_encode(['uuid' => $uuid]),
+            'raw',
+            [],
+            false
+        );
```

Comment:
Probably not for now, but would be nice to have a shortcut method for making clientsecret API calls without having to specify all the parameters like this.

Reply:
Mmmm yeah - would need to jig the method signature around as it would only remove the need for the last two arguments. Will keep in mind next time we're looking at rest client 👍